### ISSUE TITLE
no-deprecated-classes misses deprecated classes with variant prefixes

### ISSUE THREAD

**Issue by chrisclapham-eightcap** [schoero/eslint-plugin-readable-tailwind#372]
`no-deprecated-classes` correctly flags bare deprecated classes like `outline-none`, but misses the same class when used with a Tailwind variant prefix.

**Example:**

```tsx
// ✅ Flagged correctly
<div className="outline-none" />

// ❌ Not flagged — should be
<div className="focus:outline-none" />
<div className="hover:outline-none" />
<div className="sm:outline-none" />
```

Expected: `focus:outline-none` should report the same deprecation as bare `outline-none` (suggesting `focus:outline-hidden`).

Actual: no diagnostic is reported.

Likely affects all deprecated classes when combined with any variant prefix (`focus:`, `hover:`, `sm:`, `dark:`, stacked variants like `sm:focus:`, etc.).

**Versions:**
- `eslint-plugin-better-tailwindcss`: 0.1.3
- `tailwindcss`: 4.2.4
- `eslint`: 9.39.4
- Node: 22

**Comment by schoero** [comment]
The class `outline-none` from your example is not deprecated.

The rule `no-deprecated-classes` should already handle variants correctly:

<img width="1708" height="386" alt="Image" src="https://github.com/user-attachments/assets/6d2873ac-bdb2-41cc-a5e3-d8b9d653fb37" />

Could you double-check which plugin version you’re using and consider updating? You mentioned v0.1.3, but that version didn’t include this rule yet.
